styles/helpers/_text.scss

```diff
@@ -1,3 +1,9 @@
+.text-inherit {
```

Comment:
This new helper is handy when I want to set standards on a container element e.g. `Meta.Heading` and I don't want to have to set the standards every time I want to use a `h2` or `h3` etc. See https://github.com/gathercontent/gather-ui/pull/884/files#diff-68ce1da9d41e727fc2bf0006559c0562d0a722bcd741b166f852187a751a8d26R36 for a example.